### DISCUSSION TITLE
Lost all my channels- please help

### DISCUSSION THREAD

**Discussion by homenode80** [lightningnetwork/lnd#7536] [Troubleshooting]
Hi guys,

When I updated the system on my Raspiblitz, all my channels (about 20) have disappeared and they don't show up anywhere on RTL or the Raspiblitz app.

Can anyone please help me as I have lost all the BTC that I had put on Lightning....?

My pubkey is:

035bddb48a2b891a8e2a704234cb13405d63c62fcd8aa1c6f5ea10c4c027654af7

**Comment by guggero** [comment]
Sounds like the `channel.db` is in a wrong place where `lnd` doesn't recognize it. Do you see none of the channels in `lncli pendingchannels` or `lncli closedchannels`?
As long as you have a `channel.backup` file somewhere, you haven't lost your money. But chances are you haven't lost the channels either, probably just a file system issue (something not mounted or not migrated). The folks over at RaspiBlitz might be able to help you further, since it might be something depending on the version you updated from or to.

**Comment by homenode80** [comment]
Thanks. Well those two commands show either 0 or (), so nothing seems to show up.
The problem is I can't find the backup on my computer...
This happened after I upgraded to the new bitcoin software a few months ago, not Raspiblitz upgrade.
Is there anything else I could do...?

**Reply by guggero** [reply]
You'll need to look around on the device and see if you can either find the `channel.db` or `channel.backup` file. It's normally somewhere in `/mnt/hdd/lnd` (I think). Maybe there's an issue with the hard drive not being mounted correctly? There are telegram channels for support on RaspiBlitz, I'm sure they can give you more customized support (I'm not too familiar with the latest version).
